src/01-dinosaur-facts.js

```diff
@@ -22,7 +22,19 @@ const exampleDinosaurData = require("../data/dinosaurs");
  *  getLongestDinosaur(dinosaurs);
  *  //> { Brachiosaurus: 98.43 }
  */
-function getLongestDinosaur(dinosaurs) {}
+ function getLongestDinosaur(dinosaurs) {
+  let dino = {}
+  if (dinosaurs.length > 1) {
+    let length = dinosaurs[0].lengthInMeters
+    let name = dinosaurs[0].name
+    for (let  i = 0 ; i < dinosaurs.length; i++){
+      if (dinosaurs[i].lengthInMeters > length){
+        length = dinosaurs[i].lengthInMeters
+        name = dinosaurs[i].name
+      }
+    } dino[name] = length * 3.281
+  }return dino
+}
```

Comment:
Well done!

I would highly recommend watching your code placement and indentation—placing code right after a bracket like that can make it very difficult ot tell what scope a line occurs in.